EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -92,6 +92,34 @@ public struct MediaPlayerView: View {
             .tint(color)
             .wrapInButton(action: { viewModel.onTapAction?(control) })
     }
+
+    private func timeFormat(_ timeInSeconds: Float64) -> String {
+        let timeInSeconds = Int(timeInSeconds)
+        let hours:Int = Int(timeInSeconds / 3600)
+        let minutes:Int = Int(timeInSeconds % 3600 / 60)
+        let seconds:Int = Int((timeInSeconds % 3600) % 60)
+
+        if Int(viewModel.playerDuration / 3600) > 0 {
+            return String(format: "%i:%02i:%02i", hours, minutes, seconds)
+        } else {
+            return String(format: "%02i:%02i", minutes, seconds)
+        }
+    }
```

Comment:
This function doesn't depend on anything within the struct. So my suggestion is to move it to a utils file to make it more testable

Reply:
btw the Android implementation might help you, [check it out](https://github.com/profusion/android-enhanced-video-player/blob/main/androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/DateUtils.kt)